hopsworks-api/src/main/java/io/hops/hopsworks/api/filter/ProjectAuthFilter.java

```diff
@@ -105,17 +105,25 @@ public void filter(ContainerRequestContext requestContext) {
     Set<String> rolesSet;
     rolesSet = new HashSet<>(Arrays.asList(rolesAnnotation.value()));
 
-    Users user = jWTHelper.getUserPrincipal(requestContext);
-    if (requestContext.getSecurityContext().getUserPrincipal() == null && user == null) {
+    if (requestContext.getSecurityContext().getUserPrincipal() == null) {
       LOGGER.log(Level.WARNING, "Authentication not done. No user found.");
```

Comment:
@ErmiasG if the principal is null, it means the user is not logged in? If so, seems like a DEBUG message.

Reply:
I made it warning b/c if someone forgets to add the jwt or api annotation they will see it in the logs, we do not enable debug log by default. This should not happen in production.